### ISSUE TITLE
508-defect-3 [SCREENREADER]: Decorative gear icons should not be announced to screen reader users

### ISSUE THREAD

**Issue by joshkimux** [department-of-veterans-affairs/va.gov-team#23437]
# [508-defect-3](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-2)

## Feedback framework

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Definition of done

1. Review and acknowledge feedback.
1. Fix and/or document decisions made.
1. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

## Point of Contact

**VFS Point of Contact:** Josh

## User Story or Problem Statement

As a screen reader user, I don't want to hear alt text of decorative images. 

## Details

The gear icons on the connected apps page are (assumably) decorative, yet have alt text:
example: "DSVAShin-2021-02-15T22:53:50.300Z logo"

This doesn't provide any additional useful information to a screen reader user and is verbose.

## Acceptance Criteria

- [ ] Alt text is removed using a blank alt e.g. `alt=""`

## Steps to Recreate

User 36, connected apps page, inspect the gear icon

## Proposed Solution (if known)

Use a blank alt e.g. `alt=""`

## WCAG or Vendor Guidance (optional)

https://www.w3.org/WAI/tutorials/images/decorative/

## Screenshots or Trace Logs

<img width="1292" alt="Screen Shot 2021-04-19 at 12 37 53 PM" src="https://user-images.githubusercontent.com/14154792/115273177-3cb3df80-a10d-11eb-9390-9d8b6a549392.png">

**Comment by erikphansen** [comment]
@joshkimux is using an `aria-hidden="true"` on the `img` also a valid approach to this?

**Comment by erikphansen** [comment]
@tmitchellgcio we should leave this open and let Josh or Angela close it after they've validated the fix in staging.

**Comment by tmitchellgcio** [comment]
Sorry about that. No problem

**Comment by joshkimux** [comment]
A blank alt will do the trick without the need for `aria-hidden="true"`
I'll validate in staging now 😄

**Comment by joshkimux** [comment]
<img width="1556" alt="Screen Shot 2021-07-06 at 12 24 53 PM" src="https://user-images.githubusercontent.com/14154792/124635293-43710980-de55-11eb-8e7c-8d291f42c952.png">

Alt should be blank here `alt=""`; that will allow the screen reader to skip it.

**Comment by tmitchellgcio** [comment]
> <img alt="Screen Shot 2021-07-06 at 12 24 53 PM" width="1556" src="https://user-images.githubusercontent.com/14154792/124635293-43710980-de55-11eb-8e7c-8d291f42c952.png">
> 
> Alt should be blank here `alt=""`; that will allow the screen reader to skip it 👍

Odd it should be alt="" I will do some more checking on my end, but I removed the code that should be showing that text in the alt tag

**Comment by tmitchellgcio** [comment]
@joshkimux is the latest master in staging?

**Comment by joshkimux** [comment]
Just checked, alt is blank now 👍 
Great work y'all 🥳
